packages/ui/src/composables/useEvent.ts

```diff
@@ -28,12 +28,12 @@ export const useEvent = <N extends UseEventEventName, E extends Event>(
 
   watch(source, (newValue, oldValue) => {
     if (!Array.isArray(event)) {
-      unref(newValue)?.addEventListener(event, listener as any, capture)
-      unref(oldValue)?.removeEventListener(event, listener as any, capture)
+      unref(newValue)?.addEventListener?.(event, listener as any, capture)
```

Comment:
How is that possible for html element to not have addEventListener?

If this is vnode, you might need useHTMLElement to do vnode.$el

Reply:
Because of this: 
<img width="600" alt="image" src="https://user-images.githubusercontent.com/29167241/201029841-02134794-520a-4600-b17f-f8e030a5c4f8.png">